selfservice/flow/registration/flow.go

```diff
@@ -97,10 +97,19 @@ type Flow struct {
 
 	// CSRFToken contains the anti-csrf token associated with this flow. Only set for browser flows.
 	CSRFToken string    `json:"-" db:"csrf_token"`
-	NID       uuid.UUID `json:"-"  faker:"-" db:"nid"`
+	NID       uuid.UUID `json:"-" faker:"-" db:"nid"`
 
 	// TransientPayload is used to pass data from the registration to a webhook
-	TransientPayload json.RawMessage `json:"transient_payload,omitempty"  faker:"-" db:"-"`
+	TransientPayload json.RawMessage `json:"transient_payload,omitempty" faker:"-" db:"-"`
+
+	// Contains a list of actions, that could follow this flow
+	//
+	// It can, for example, contain a reference to the verification flow, created as part of the user's
+	// registration.
+	//
+	// required: false
+	// swagger:ignore
```

Comment:
Since we aren't returning the registration flow itself, once the registration finishes, this is just used internally, and should not be sent to the client.

Reply:
Remove the required field, and swagger ignore. `json:"-"` already omits it from swagger

Reply:
Done.